docs/machine-learning/how-to-guides/index.md

```diff
@@ -8,6 +8,33 @@ ms.date: 10/05/2018
 In the How to section of the ML.NET Guide you can find quick answers
 to common questions. In some cases, articles may be listed in multiple sections to make them easy to find.
 
+## ML.NET
```

Comment:
Since the header of this section is ML.NET, is it necessary for all but one item in the section to end with "- ML.NET"?

Reply:
nope, I agree.